### PR TITLE
Refactor oapi fetch task

### PR DESCRIPTION
**Pull request by Vachman**
Fixes oapi rake task API secs generation tasks. At the moment it generates specs for only one swagger-doc path.

### CODE REVIEW

**Comment by Vachman** [comment]
Rubocop is failing on files which i didn't touch. Any idea how can this be solved?

**Comment by LeFnord** [comment]
run `bundle exec rubocop --auto-gen-config` … will have a look on it afterwards

**Review by LeFnord — COMMENTED** [review]
Is the README still up to date?

**Comment by Vachman** [comment]
> run `bundle exec rubocop --auto-gen-config` … will have a look on it afterwards

Hey, this is the result:

Снимок экрана 2022-01-13 в 14 35 51

**Comment by LeFnord** [comment]
> Rubocop is failing on files which i didn't touch.

it seems most comes from your changes

File: .rubocop_todo.yml [hunk 1]
```diff
@@ -35,6 +41,20 @@ Style/ClassVars:
...
+  Exclude:
+    - 'spec/lib/endpoint_spec.rb'
+    - 'spec/lib/version_spec.rb'
+    - 'spec/support/mock_parser.rb'
+    - 'spec/support/model_parsers/mock_parser.rb'
+    - 'spec/swagger_v2/api_swagger_v2_hide_documentation_path_spec.rb'
+    - 'spec/swagger_v2/api_swagger_v2_mounted_spec.rb'
+    - 'spec/swagger_v2/api_swagger_v2_spec.rb'
+    - 'spec/swagger_v2/errors_spec.rb'
+    - 'spec/swagger_v2/reference_entity_spec.rb'
```
Comment (LeFnord) [inline .rubocop_todo.yml:60]: 👍

**Review by LeFnord — APPROVED** [review]
thanks @Vachman

**Comment by Vachman** [comment]
Hey @LeFnord, 
could it be possible to merge this feature or it need more tuning?

**Comment by LeFnord** [comment]
sure and thanks again

File: lib/grape-swagger/rake/oapi_tasks.rb [hunk 2]
```diff
@@ -102,8 +115,15 @@ def error?
...
-      def file
-        name = ENV['store'] == 'true' || ENV['store'].blank? ? 'swagger_doc.json' : ENV['store']
+      def file(url)
+        api_version = url.split('/').last
+
+        name = if ENV['store'] == 'true' || ENV['store'].blank?
+                 "swagger_doc_#{api_version}.json"
+               else
+                 ENV['store'].sub('.json', "_#{api_version}.json")
+               end
```
Comment (x-yuri) [inline lib/grape-swagger/rake/oapi_tasks.rb:125]: looks like a breaking change to me, which made it into v1.5.0

Comment (yamanaltereh) [reply lib/grape-swagger/rake/oapi_tasks.rb:125]: what is the reason of this change? is it really necessary?

it's really weird change, passing ENV['store'] and getting different file name at the end!
example below:

```
bundle exec rails oapi:fetch store=public/name.json
```
and getting `public/name_swagger_doc.json`

**Pull request merged**